Refactor docs content for an all org Neon (#3455)
neondatabase/website: content/docs/manage/accounts.md

---
title: Accounts
subtitle: Manage your Neon account
enableTableOfContents: true
updatedOn: '2025-04-17T08:58:53.655Z'
---

Your **Neon account** is your personal identity for logging in, managing your profile, and authenticating actions across all organizations you belong to.

## Account settings

You can access your Neon account settings from anywhere in the Console. Just click your profile avatar and select **Account settings** from the menu.

![Open the profile menu and select Account settings.](/docs/manage/profile_dropdown.png)

> Here's what you can do from **Account settings**.

Reviewer: Given that we're saying "Here's what you can do", should the title of each numbered section have a verb?

Update, Change, Create, Enable... etc.

Your name and email come from your social account. Feel free to change your name to whatever works for you. If you change your email, we'll unlink your social account and switch you to email sign-in. After that, you'll use your new email and password to log in.

**Changing your email**

If you change your email (whether you started with email or social login), you'll get a verification email to confirm. Once confirmed, your new address becomes your login. If you're using a social login and change your email, we'll unlink your social account and switch you to email sign-in.

[Change your email in the Neon Console](https://console.neon.tech/app/settings?modal=change_email)

### Need to switch your login method?

**If you signed up with Google or GitHub**, you can switch to email login by changing your email and setting a password.

**If you signed up with email,** it's not currently possible to switch to a social login. [Contact Support](/docs/introduction/support) and we'll help you out.

## Change password

No surprises here — just enter your current password, then your new one (twice). We'll enforce our current password rules for your security.

## Personal API keys

> Personal API keys let you securely access Neon from the command line, scripts, or third-party tools. Your personal API key works for any organization you belong to — so you can manage projects, automate tasks, or use integrations across all your orgs with a single key. The actions you can perform with your key depend on your role in each org (admin, member, or collaborator). You can create, view, and revoke your personal API keys here in your account settings.

Reviewer: ```suggestion
Personal API keys let you securely interact with the [Neon API](https://api-docs.neon.tech/reference/getting-started-with-neon-api), including through command-line tools, scripts, or third-party integrations that use the API.. Your personal API key works for any organization you belong to — so you can manage projects, automate tasks, or use integrations across all your orgs with a single key. The actions you can perform with your key depend on your role in each org (admin, member, or collaborator). You can create, view, and revoke your personal API keys here in your account settings.
```

[Learn more about API keys](/docs/manage/api-keys)

![Personal API keys section in Account settings.](/docs/manage/personal_api_key.png)

## Early access

> Enable Early Access for your Neon account to preview new features before they go live.

Reviewer: It would be nice to have a screen cap here to provide more context. 
<img width="819" alt="image" src="https://github.com/user-attachments/assets/4ea5dcc7-1a88-4cce-becf-cc89f6ecf02f" />

Author: Looks like this is now removed from Account settings. There is no account-level early access, just at the org-level.
